### ISSUE TITLE
Default serializer options ignored

### ISSUE THREAD

**Issue by j-mcnally** [rails-api/active_model_serializers#898]
previously i could

```
def default_serializer_options
        {root: true}
end
```

This seems to have been removed, is there a way in master to always include root?

**Comment by j-mcnally** [comment]
fwiw, i fixed it for now like this

```
class RootAdapter < ActiveModel::Serializer::Adapter::Json
  def initialize(serializer, options = {})
    super
    serializer.root = true
  end
end
```

**Comment by joaomdmoura** [comment]
@j-mcnally what is the version of AMS you are using?

**Comment by j-mcnally** [comment]
Master

**Comment by Rodrigora** [comment]
Hi @j-mcnally, 

(according this)[https://github.com/rails-api/active_model_serializers#flattenjson], if you want to generate
the responses with root key, you should use `Json` adapter. 

By default, AMS uses `FlattenJSON` which doesn't generates the root key.

Did you configure the Json adapter for you application?

**Comment by joaomdmoura** [comment]
Indeed @j-mcnally, as @Rodrigora mentioned, back then it wasn't true, but right now only the `Json` adapter enables you to use root.
I'm closing this one but let me know in case you have any problems with it.

**Comment by iRet** [comment]
So, guys, what is the default_serializer_options destiny? is it available, renamed or removed?

**Comment by joaomdmoura** [comment]
@iRet removed

**Comment by iRet** [comment]
@joaomdmoura is any convenient way to specify to serializers options globally exist? instead of every time to `render`.

**Comment by joaomdmoura** [comment]
that depends on the option.
A lot of the usual options are related to the adapter you choose, so it's a way of set a convention application-wide without bother using it on every `render`

**Comment by iRet** [comment]
@joaomdmoura for example if I want to specify some context to Serializer. I think it is useful to have at least two sets of fields one light for index and another detailed. I know I can implement two separate serializer and I've used this approach but finally found it is more conveninent to tweak one.

**Comment by joaomdmoura** [comment]
Other option would be to use the `field` option on the render.
But indeed, if are going to use the **same** different set of attributes on many actions (rather than just `index`, as the example above), the only way to make it DRY right now is to have two serializers.
I agree that have different set of fields might be convenient, but seems to be an edge-case for me.

**Comment by paulief** [comment]
@joaomdmoura I apologize if this is the wrong place for this, but I didn't want to create a duplicate issue. I can create a new issue if that's the right way.

I am trying to make the move from 0.9.3 -> 0.10.x and have the following issue:

In 0.9.3, I used the `:attributes` adapter which would include deeply nested associations by default. On upgrading to 0.10.3, it no longer does so I went to switch to the `:json` adapter. With this adapter, I need to set `include: { field1: [:nested1, :nested2] }` or just `include: '**'` on many of my calls to `render` in order to get the nested associations. Also, I do not want the root set since the front-end app expects the JSON with no root except for collections. I can't seem to get `root: false` to work, but it seems like I'd need to set that everywhere too.

Since `default_serializer_options` has been deprecated, is there another way to accomplish this? Also, is `root: false` still functional in 0.10.3 for the `:json` adapter?
